runtime/moonbase/src/weights/pallet_xcm.rs

```diff
@@ -346,4 +346,14 @@ impl<T: frame_system::Config> pallet_xcm::WeightInfo for WeightInfo<T> {
 			.saturating_add(T::DbWeight::get().reads(2_u64))
 			.saturating_add(T::DbWeight::get().writes(1_u64))
 	}
+
+    fn add_authorized_alias() -> Weight {
+        // TODO: regenerate this file
+        Weight::from_parts(100_000, 0)
```

Comment:
Lets try to generate this locally, just to be sure the benchmarks are not broken